source/includes/_under-the-hood.html.md

```diff
@@ -1,8 +1,8 @@
-## Under the Hood
+# Under the Hood
```

Comment:
It seems we could flatten this hierarchy a bit and make major components, versioning, etc go up a level.  Then just eliminate under the hood and use the sentence below as an intro (or in addition to eliminating "Under the hood" also eliminate this intro sentence)